packages/core/src/components/axes/two-dimensional-axes.ts

```diff
@@ -40,11 +40,13 @@ export class TwoDimensionalAxes extends Component {
 		} else {
 			this.model.getOptions().axes = {
 				left: {
-					primary: true
+					primary: true,
+					includeZero: true
 				},
 				bottom: {
 					secondary: true,
-					type: this.model.getDisplayData().labels ? ScaleTypes.LABELS : undefined
+					type: this.model.getDisplayData().labels ? ScaleTypes.LABELS : undefined,
+					includeZero: true
 				}
 			};
 
```

Comment:
Are we able to eliminate having to do this by performing the logic inside of `mergeDefaultChartOptions`?

Reply:
We can do it easily by setting these as default axes configuration and then if axes are not provided take the left and bottom default ones. 
The problem is that for set `type` (that should be `scaleType`) we don't have the model.

Also, we can remove these lines
https://github.com/carbon-design-system/carbon-charts/blob/535378fb4c02a7fd9bddb5dcca5d134703428d17/packages/core/src/components/axes/two-dimensional-axes.ts#L32-L37

Reply:
Oh right. We should rename `type` to `scaleType` (we should also probably wrap the object to the correct TS type so it would shout if we mess it up down the road)

Anything we can remove out of this file in terms of setting the correct defaults and move into the `CartesianScales` service would be great!